### PR TITLE
added an option to batch save to kv store

### PR DESCRIPTION
**Pull request by esharf**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/35509

## Description
added the ability to use a JSON format list to upload more than one entry at a time.

## Does it break backward compatibility?
   - [ ] Yes
   - [x] No

## Must have
- [x] Tests

### CODE REVIEW

File: Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.py [hunk 1]
```diff
@@ -954,11 +954,14 @@ def get_last_update_in_splunk_time(last_update):
     """
     last_update_utc_datetime = dateparser.parse(last_update, settings={'TIMEZONE': 'UTC'})
     params = demisto.params()
-    splunk_timezone = params.get('timezone')
-    if not splunk_timezone:
+
+    try:
+        splunk_timezone = int(params['timezone'])
+    except (KeyError, ValueError):
```
Comment (yaakovi) [inline Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.py:960]: is this related to another issue?

**Review by yaakovi — APPROVED** [review]

File: Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.py (hunk 1 above)
Comment (esharf) [reply Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.py:960]: no.
it was just mypy.
